### ISSUE TITLE
Add custom filter or realm

### ISSUE THREAD

**Issue by harshmehta5** [johntostring/spring-boot-shiro#1]
Can you give an example of how to add a custom filter or realm using the application.yml file

**Comment by zhangjinhang** [comment]
Sorry about that it does not support custom Filter(Shiro managed) so far.

Just set custom realm like this:

```
shiro:
  realm-class: #full.class.name.to.YourRelam
```

but if you set this at the same time,  your custom realm will not work.

```
shiro:
  realm:
    jdbc:
      enabled: true
```

If you really need a custom Filter (Shiro managed). I will work on it these days, or maybe you can commit a Pull Request.

**Comment by johntostring** [comment]
closed #2
